db/db.js

```diff
@@ -249,11 +249,11 @@ async function updateSessionState(sessionid, state, clientParam) {
 }
 
 // Updates the value of the alert flag in the location database
-async function updateSentAlerts(locationid, sentalerts, clientParam) {
+async function updateSentAlerts(locationid, sentalerts, component, clientParam) {
   try {
     const results = await runQuery(
       'updateSentAlerts',
-      'UPDATE locations SET heartbeat_sent_alerts = $1 WHERE locationid = $2 RETURNING *',
+      `UPDATE locations SET ${component}_heartbeat_sent_alerts = $1 WHERE locationid = $2 RETURNING *`,
```

Comment:
This feels like opening us up to SQL injection. I'd prefer an if-else here or two separate DB functions.